Gems/OpenXRVk/Code/Source/Platform/Common/Default/OculusTouch_Default.cpp

```diff
@@ -0,0 +1,162 @@
+
+        using xrc = InputDeviceXRController;
+        m_xrPathMap = {
+            { xrc::Button::A, "/user/hand/right/input/a/click" },
+            { xrc::Button::B, "/user/hand/right/input/b/click" },
+            { xrc::Button::X, "/user/hand/left/input/x/click" },
+            { xrc::Button::Y, "/user/hand/left/input/y/click" },
+            { xrc::Button::Home, "/user/hand/right/input/system/click" },
+            { xrc::Button::Menu, "/user/hand/left/input/menu/click" },
+            { xrc::Button::L3, "/user/hand/left/input/thumbstick/click" },
+            { xrc::Button::R3, "/user/hand/right/input/thumbstick/click" },
+            { xrc::Button::TA, "/user/hand/right/input/a/touch" },
+            { xrc::Button::TB, "/user/hand/right/input/b/touch" },
+            { xrc::Button::TX, "/user/hand/left/input/x/touch" },
+            { xrc::Button::TY, "/user/hand/left/input/y/touch" },
+            { xrc::Button::TLStick, "/user/hand/left/input/thumbstick/touch" },
+            { xrc::Button::TRStick, "/user/hand/right/input/thumbstick/touch" },
+            { xrc::Button::TLRest, "/user/hand/left/input/thumbrest/touch" },
+            { xrc::Button::TRRest, "/user/hand/right/input/thumbrest/touch" },
+            { xrc::Button::TLTrig, "/user/hand/left/input/trigger/touch" },
+            { xrc::Button::TRTrig, "/user/hand/right/input/trigger/touch" },
+            { xrc::Trigger::LTrigger, "/user/hand/left/input/trigger/value" },
+            { xrc::Trigger::RTrigger, "/user/hand/right/input/trigger/value" },
+            { xrc::Trigger::LGrip, "/user/hand/left/input/squeeze/value" },
+            { xrc::Trigger::RGrip, "/user/hand/right/input/squeeze/value" },
+            { xrc::ThumbStickAxis1D::LX, "/user/hand/left/input/thumbstick/x" },
+            { xrc::ThumbStickAxis1D::LY, "/user/hand/left/input/thumbstick/y" },
+            { xrc::ThumbStickAxis1D::RX, "/user/hand/right/input/thumbstick/x" },
+            { xrc::ThumbStickAxis1D::RY, "/user/hand/right/input/thumbstick/y" },
+            { xrc::ControllerPosePosition::LPos, "/user/hand/left/input/grip/pose" },
+            { xrc::ControllerPosePosition::RPos, "/user/hand/right/input/grip/pose" },
+            { xrc::ControllerPoseOrientation::LOrient, "/user/hand/left/input/aim/pose" },
+            { xrc::ControllerPoseOrientation::ROrient, "/user/hand/right/input/aim/pose" },
+        };
+    }
+
+    ////////////////////////////////////////////////////////////////////////////////////////////////
+    InputDeviceOculusTouch::~InputDeviceOculusTouch() = default;
+
+    ////////////////////////////////////////////////////////////////////////////////////////////////
+    AZStd::string_view InputDeviceOculusTouch::GetInputChannelPath(const InputChannelId& channelId) const
+    {
+        if (const auto it = m_xrPathMap.find(channelId);
+            it != m_xrPathMap.end())
+        {
+            return it->second;
+        }
+        return {};
```

Comment:
It's not very clear what `return {};` is for `string_view`, is it an invalid view? is it pointing to empty string?
Maybe to be clear this can be `return {""};` instead?

Reply:
I'll be changing it to return an `AZStd::string` based on previous feedback, but the `{}` in this instance just means default constructed, and for `basic_string_view` both members `const_pointer m_begin` and `size_type m_size` are also initialized in [the same way](https://en.cppreference.com/w/cpp/language/list_initialization#direct-list-initialization).